framework/src/meshgenerators/MeshDiagnosticsGenerator.C

```diff
@@ -335,6 +343,66 @@ MeshDiagnosticsGenerator::checkWaterTightSidesets(const std::unique_ptr<MeshBase
   diagnosticsLog(message, _check_watertight_sidesets, num_faces_without_sideset);
 }
 
+void
+MeshDiagnosticsGenerator::checkWatertightNodesets(const std::unique_ptr<MeshBase> & mesh) const
+{
+  /*
+  Diagnostic Overview:
+  1) Mesh precheck
+  2) Loop through all elements
+  3) Loop through all sides of that element
+  4) If side is external loop through its nodes
+  5) If node is not associated with any nodeset add to list
+  6) Print out node id
+  */
+  if (mesh->mesh_dimension() < 2)
+    mooseError("The nodeset check only works for 2D and 3D meshes");
+  auto & boundary_info = mesh->get_boundary_info();
+  boundary_info.build_side_list();
```

Comment:
What's going on here?  This is the `build_side_list()` variant that returns a vector of tuples, but if nothing uses the return value then it's just an expensive no-op.

Reply:
I think I copied that over from a similar function I made previously and then forgot to remove it. I'l fix that real quick.